daliuge-translator/test/reproducibility/reproGraphs/misc.graph

```diff
@@ -53,6 +53,37 @@
             "x": 220.2871317489371,
             "y": 488.67884359131426
         },
+        {
```

Comment:
@awicenec was this commit intended for this branch?

Reply:
Yes, because we have deprecated the Description node. Description now is not a special node on the graph, but part of the graph metadata.

Reply:
Makes sense. If you're happy with these changes then I will merge.